File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;

    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);

    int cents = round(dollars * 100);

    int coins = 0;
    int change_owed = cents;
    while (change_owed >= 25)
    {
        coins = coins + 1;
        change_owed = change_owed - 25;
    }
    while (change_owed >= 10)
    {
        coins = coins + 1;
        change_owed = change_owed - 10;
    }
    while (change_owed >= 5)
    {
        coins = coins + 1;
        change_owed = change_owed - 5;
    }
    while (change_owed >= 1)
    {
        coins = coins + 1;
        change_owed = change_owed - 1;
    }

    printf("%i", coins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    dollars = get_float("Change owed: ")
    if dollars > 0:
        break

cents = round(dollars * 100);

coins = 0
change_owed = cents
while change_owed >= 25:
    coins = coins + 1
    change_owed = change_owed - 25

while change_owed >= 10:
    coins = coins + 1
    change_owed = change_owed - 10

while change_owed >= 5:
    coins = coins + 1
    change_owed = change_owed - 5

while change_owed >= 1:
    coins = coins + 1
    change_owed = change_owed - 1

print(coins)
